api/controllers/node.js

```diff
@@ -45,11 +45,11 @@ function NodeController(scope) {
 /**
  * Description of the function.
  *
- * @param {Object} context
+ * @param {Object} _context
  * @param {function} next
  * @todo Add description for the function and the params
  */
-NodeController.getConstants = function(context, next) {
+NodeController.getConstants = function(_context, next) {
```

Comment:
I raised same question before, I am unable to understand why this name change occurs in one controller here and not on all other controllers.

Reply:
Please check this discussion https://github.com/LiskHQ/lisk/pull/2589#discussion_r237447209

Reply:
I don't feel confident if that's something defined by any official language style guide e.g. https://www.python.org/dev/peps/pep-0008/

But since its been forced by `es-lint` we can move forward with it.